components/common-data-items.js

```diff
@@ -264,6 +266,30 @@ export function BiosampleDataItems({
           </DataItemValue>
         </>
       )}
+      {pooledFrom.length > 0 && (
+        <>
+          <DataItemLabel>Biosample(s) Pooled From</DataItemLabel>
+          <DataItemValue>
+            <SeparatedList>
+              {pooledFrom.map((biosample) => (
+                <Link href={biosample["@id"]} key={biosample.uuid}>
+                  {biosample.accession}
+                </Link>
+              ))}
+            </SeparatedList>
+          </DataItemValue>
+        </>
+      )}
+      {partOf && (
+        <>
+          <DataItemLabel>Part of Biosample</DataItemLabel>
+          <DataItemValue>
+            <Link href={partOf["@id"]} key={partOf.uuid}>
```

Comment:
You don’t need a key for the link. `<DataItemValue>` is just a simple wrapper. `<SeparatedList>` and `<SearchListItemMeta>` need keys because they have internal loops.